protos/mocap/mocap.proto

```diff
@@ -0,0 +1,131 @@
+syntax = "proto3";
+
+package mavsdk.rpc.mocap;
+
+option java_package = "io.mavsdk.mocap";
+option java_outer_classname = "MocapProto";
+
+/**
+ * TODO
+ */
+service MocapService {
+    /*
+     * TODO
+     */
+    rpc CreateVisionPositionEstimateStream(stream VisionPositionEstimateRequest) returns(stream VisionPositionEstimateResponse) {}
+    rpc CreateAttPosMocapStream(stream AttPosMocapRequest) returns(stream AttPosMocapResponse) {}
+    rpc CreateOdometryStream(stream OdometryRequest) returns(stream OdometryResponse) {}
+}
+
+message VisionPositionEstimateRequest {
+    VisionPositionEstimate vision_position_estimate = 1;
+}
+message VisionPositionEstimateResponse {
+  MocapResult mocap_result = 1;
+}
+
+// Global position/attitude estimate from a vision source.
+message VisionPositionEstimate {
+    uint64 time_usec = 1; // Position frame timestamp UNIX Epoch time (0 to use Backend timestamp)
+    float x = 2; // Global X position (m)
+    float y = 3; // Global Y position (m)
+    float z = 4; // Global Z position (m)
+    float roll = 5; // Roll angle (rad)
+    float pitch = 6; // Pitch angle (rad)
+    float yaw = 7; // Yaw angle (rad)
+    repeated float covariance = 8; // Row-major representation of pose 6x6 cross-covariance matrix upper right triangle. Set first to NaN if unknown.
+    uint32 reset_counter = 9; // Estimate reset counter.
+}
+
+
+message AttPosMocapRequest {
+    AttPosMocap att_pos_mocap = 1;
+}
+message AttPosMocapResponse {
+  MocapResult mocap_result = 1;
+}
+
+// Motion capture attitude and position
+message AttPosMocap {
```

Comment:
I would call it AttitudePositionMocap. In general I vote for more verbose but clearer naming.